Sources/GraphQLTransportWS/Server.swift

```diff
@@ -6,7 +6,7 @@ import GraphQLRxSwift
 import NIO
 import RxSwift
 
-/// Server implements the server-side portion of the protocol, allowing a few callbacks for customization.
+/// Server implements the server-side portion of the protocol, allowing a few callbacks for customization. 0 or 1 subscriptions per connection and no more.
```

Comment:
Why do we specify 0 or 1 subscription? I believe it can support any number of subscriptions, as long as they have different IDs.

Reply:
This was just an incorrect assumption by me that I forgot to remove. Removed now.